src/components/Search.tsx

```diff
@@ -0,0 +1,21 @@
+import { useState } from "react";
+
+interface Props {
+  intitialSearchText: string;
+  onSearch: (searchText: string) => void;
+}
+export default function Search({ intitialSearchText, onSearch }: Props) {
+  const [searchText, setSearchText] = useState(intitialSearchText);
+
+  return (
+    <div>
+      <input
```

Comment:
Its a good practive to provide the placeholder for inputs - the initialSearchText can be an empty string for example.

Reply:
Added placeholder text